cmd/seed.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"os"
+	"os/signal"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/seed"
+)
+
+var (
+	seedCmd = &cobra.Command{
+		GroupID: groupLocalDev,
+		Use:     "seed",
+		Short:   "Generate seed data",
```

Comment:
Shall we move this under `supabase gen seed` instead since it generates a seed file?

If we continue using `supabase seed`, my expectation would be to actually seed the local / remote database.